cmd/di.go

```diff
@@ -681,8 +686,12 @@ func (di *Dependencies) bootstrapNetworkComponents(options node.Options) (err er
 
 	di.MysteriumAPI = mysterium.NewClient(di.HTTPClient, network.MysteriumAPIAddress)
 
-	log.Info().Msg("Using Eth endpoint: " + network.EtherClientRPC)
+	di.BrokerConnection, err = nats.NewConnection(di.NetworkDefinition.BrokerAddress)
```

Comment:
This `err` is never checked, you are replacing it in the next line.

Reply:
Fixed